src/alfasim_sdk/result_reader/aggregator.py

```diff
@@ -85,6 +89,12 @@ class ResultsNeedFullReloadError(RuntimeError):
     """
 
 
+@attr.s(slots=True, hash=False)
+class UQResultMetadata:
+    global_gsa_coefficients = attr.ib(validator=attr.validators.instance_of(dict))
+    result_directory = attr.ib(validator=attr.validators.instance_of(Path))
```

Comment:
- [ ] Type annotations please:

```suggestion
@attr.s(slots=True, hash=False)
class UQResultMetadata:
    global_gsa_coefficients: dict[???, ???] = attr.ib(validator=attr.validators.instance_of(dict))
    result_directory: Path = attr.ib(validator=attr.validators.instance_of(Path))
```